Backend/core/comms.py

```diff
@@ -226,3 +245,6 @@ def start_comms():
     vps_thread.start()
     socket_thread = threading.Thread(target=lambda: telemetry.listen_udp(config.UDP_PORT))
     socket_thread.start()
+
+    # start file sync
+    sync(telemetry.fs_down_callback)
```

Comment:
Should this be in a seperate process? parsing takes significant cpu time we don't want it to block other threads (Python fake threads 👎 )

Reply:
If I move this sync() call out into its own process, would it even be possible for it to have `fs_down_callback()` in its process or would that not be possible because it's an instance method of the Telemetry class? Just wondering how much refactoring I'd need to do to achieve this

Reply:
I think the process works like fork()? so it probably should be able to access the callback. (test it)

Reply:
It does create a new process, but then seems to get stuck in a deadlock? It was waiting on Redis for 30+ seconds, but what I'm unsure about is whether the primary backend process is capable of continuing to receive data from UDP and the VPS the whole time or whether both processes get stuck at the same time.

```diff
diff --git a/Backend/core/comms.py b/Backend/core/comms.py
index e26c601..b6c025c 100644
--- a/Backend/core/comms.py
+++ b/Backend/core/comms.py
@@ -5,6 +5,7 @@ import aiohttp
 import asyncio
 import config
 
+from multiprocessing import Process
 from . import db
 from file_sync.file_sync_down.main import *
 
@@ -245,4 +246,6 @@ def start_comms():
     socket_thread.start()
 
     # start file sync
+
+    p = Process(target=sync, args=[telemetry.fs_down_callback])
     sync(telemetry.fs_down_callback)
diff --git a/Backend/file_sync b/Backend/file_sync
index ca466b8..924590d 160000
--- a/Backend/file_sync
+++ b/Backend/file_sync
@@ -1 +1 @@
```

![image](https://github.com/badgerloop-software/chase-car-dashboard/assets/17338790/769ec280-b0a6-4c65-ab27-9e8272b9b7b9)

Reply:
Doesn't seem to work with this diff:

```diff
diff --git a/Backend/core/comms.py b/Backend/core/comms.py
index e26c601..d888eef 100644
--- a/Backend/core/comms.py
+++ b/Backend/core/comms.py
@@ -5,6 +5,7 @@ import aiohttp
 import asyncio
 import config
 
+from multiprocessing import Process
 from . import db
 from file_sync.file_sync_down.main import *
 
@@ -245,4 +246,6 @@ def start_comms():
     socket_thread.start()
 
     # start file sync
-    sync(telemetry.fs_down_callback)
+
+    p = Process(target=sync, args=[teleme
```

Reply:
I know it doesn't because it never creates the Downloaded Files and downloaded_zips folders